cli/cli/command_framework/lowlevel/lowlevel_kurtosis_command_test.go

```diff
@@ -0,0 +1,31 @@
+package lowlevel
+
+import (
+	"github.com/kurtosis-tech/stacktrace"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestRemoveFilePathFromErrorMessage(t *testing.T) {
```

Comment:
Could you add a test containing a file path to check that this is removed?

Reply:
This what it is intended to do? stacktrace.New() and stacktrace.Propagate adds the file path automatically and then i remove them. @leoporoli